src/prepare.js

```diff
@@ -1,16 +1,12 @@
 const { readFile, writeFile } = require('fs').promises;
 const path = require('path');
 const execa = require('execa');
+const { VERSION_REGEX } = require('./common');
 
 const writeVersion = async ({ versionFile, nextVersion, logger, cwd }) => {
   const fullVersionPath = path.resolve(cwd, versionFile);
   const versionContents = await readFile(fullVersionPath, 'utf8');
-  const newContents = versionContents.replace(
-    // taken from https://github.com/svenfuchs/gem-release/blob/0f5f2382ef960d40169400a8aa25085cd37d26b0/lib/gem/release/files/version.rb#L7
-    /(VERSION\s*=\s*['"])(?:(?!"|').)*(['"])/,
-    // see https://guides.rubygems.org/patterns/#prerelease-gems
-    `$1${nextVersion.replace('-', '.')}$2`,
```

Comment:
I'm pretty sure (fingers crossed) that the dash will still work with rubygems (they say ` It just has to have a letter in it, and you’re set.`). I'd like to avoid the version in rubygems being different from the one in the CHANGELOG/Github.